repository/s.json

```diff
@@ -414,12 +414,14 @@
 		},
 		{
 			"name": "Scheme",
-			"details": "https://github.com/egrachev/sublime-scheme",
-			"labels": ["language syntax", "snippets"],
+			"description": "Syntax Highlighting, smart snippets, code formatting, code navigation and more for the Scheme language",
```

Comment:
You don’t have to put a description here, package control will take the description of the GitHub repo, allowing to change it at will.

Reply:
Ok. Thank you for your remind.